### DISCUSSION TITLE
Use custom subdomain with A records

### DISCUSSION THREAD

**Discussion by NiklasMerz** [community/community#22928] [Other Feature Feedback, Questions, & Ideas]
<p>I want to use my pages site with a custom subdomain like <a href="http://blog.example.com" rel="nofollow noopener">blog.example.com</a>. I have set up A records for that and it works, but the account page shows a warning that using CNAME records is recommended. Why arent A records for subdomains not recommened like for apey domains? Does this still work?</p>
<p>Thanks.</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/niklasmerz">@niklasmerz</a>,</p>
<p>Thanks for being part of the GitHub Community Forum! A records will work just fine, but CNAME records are considered best practice, so that’s why we recommend them. </p>
<p>I hope this helps!</p>
<p>Cheers!</p>

**Comment by NiklasMerz** [comment]
<p>Thanks thats fine for me. Is there any way to turn off the e-mail warning I get every time I push?</p>

**Comment by nadiajoyce** [comment]
<p>Sure thing! Check out our <a href="https://help.github.com/articles/choosing-the-types-of-notifications-you-receive/#disabling-email-service-notifications-for-pushes-to-your-repository" rel="nofollow noopener">Help article on how to do this</a>.</p>
